rules/S6370/cfamily/metadata.json

```
@@ -0,0 +1,20 @@
+{
+  "title": "Track the use of awaiter members in \"await_suspend\" after the coroutine may be resumed",
```

```suggestion
  "title": "\"await_suspend\" should not use awaiter members after the coroutine may be resumed",
```
See "Guidelines for Bug, Vulnerability, and Code Smell rules" / "Titles" [here](https://docs.sonarqube.org/latest/extend/adding-coding-rules/)

---

Ah, for whatever reason I misread the section as suggesting to use Track. thanks.